src/pages/about.astro

```diff
@@ -4,300 +4,219 @@ import CtaButtom from "../components/InlineCtaButton.astro"
-        <CtaButtom className="about-button" link="/contact">Get in Touch</CtaButtom>
-    </p>
-    <picture class="photo-right">
-        <source type="image/avif" srcset="/images/saamia.avif"/>
-		<source type="image/webp" srcset="/images/saamia.webp"/>
-        <img src="/images/saamia.jpg" alt="saamia">
-    </picture>
-    <section class="photo-right header">
-        <h3>Saamia Ahmad <span class="pronouns">(she/her)</span></h3>
-        <h4>Economist</h4>
-    </section>
-    <p class="photo-right">Saamia Ahmad is an economist at Vivic Research. Saamia respectfully acknowledges that she lives and works on the traditional territory of the Mississaugas of the Credit, the Anishnabeg, Haudenosaunee Confederacy, and the Wendat Nations. Saamia has a keen interest in public policy, with a particular emphasis on how policy can be used to improve the wellbeing of all Canadians. Saamia’s interests within economics are income inequality, shared prosperity, and advancing human welfare through progressive economics. In her previous role as a media analyst, she managed media campaigns by conducting data analyses and executing programmatic strategies to streamline the overall customer journey. In her role as economist at Vivic Research, Saamia develops economic models and designs advanced analysis projects with the intention of meaningful involvement in her projects. Saamia is a graduate of Queen’s University’s Economics department. 
-        <CtaButtom className="about-button" link="/contact">Get in Touch</CtaButtom>
-    </p>
-</div>
+	<p class='big-text'>
+		At Vivic Research, we support agents working toward social justice with economic
+		and data-driven research. We serve advocates, think tanks, governments, labour
+		unions, and other agents engaged in the development of public policy. We empower
+		our clients with economic knowledge through our practice areas: economic research,
+		data science, and training.
+	</p>
+	<div id="about-section">
+		<section>
+			<h3>
+				Our Vision
+			</h3>
+			<p>
+				Vivic Research envisions a just society, where everyone has access
+				to housing, food, safety, health and community.
+			</p>
+		</section>
+		<section>
+			<h3> 
+				Our Mission 
+			</h3>
+			<p> 
+				Vivic Research’s mission is provide alternatives to traditional economic
+				consulting. We work towards our collective liberation by actively
+				challenging standard assumptions and existing systems.
+			</p>
+		</section>
+	</div>
+
+	<h2>Our Team</h2>
+	<div class="team">
+		<section>
```

Comment:
Each person here can be under a single `<section>`, which will make styling a lot easier, and also has some visual improvements (images don't get weirdly cropped by the clip-path).